lib/Dialect/FIRRTL/Transforms/LowerBitindex.cpp

```diff
@@ -0,0 +1,156 @@
+//===- LowerBitindex.cpp - Lower Bitindex
+//-----------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the LowerBitindex pass.
+//
+//===----------------------------------------------------------------------===//
```

Comment:
It's not clear why there is a lowering pass for this.  This is directly expressible in SV with IndexedPartSelectInOutOp.